giza/giza/operations/stage.py

```diff
@@ -484,9 +478,14 @@ def sync_redirects(self, redirects):
         if not redirects:
             return
 
+        redirect_dirs = self.staging_config.redirect_dirs
+        if not redirect_dirs:
+            LOGGER.warn('No "redirect_dirs" listed for this project.')
+            LOGGER.warn('No redirects will be removed.')
+
```

Comment:
I recognize that the for key in remote_directs will ensure it won't remove stuff, but it seems weird to log:
No redirects will be removed 
and then
Finding redirects to remove.